blockkit/elements.py

```diff
@@ -717,3 +718,21 @@ def __init__(
         indent: Optional[int] = None,
     ):
         super().__init__(elements=elements, style=style, indent=indent)
+
+class FileInput(FocusableElement):
```

Comment:
I think we should inherit from the ActionableComponent class to make the tests pass.